contracts/instruments/RibbonThetaVault.sol

```diff
@@ -112,13 +123,23 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
 
         asset = _isPut ? _usdc : _asset;
         underlying = _asset;
+        collateralToken = IYearnVault(
+            _isYearnWrapped
+                ? IYearnRegistry(_yearnRegistry).latestVault(
```

Comment:
ik this expression is a little messy - solidity doesn't allow reading public immutable variables (asset) in constructor.

Reply:
It may be safer to just rely on the exact yearn vault address rather looking up the `latestVault`

Reply:
For making the statement easier to read, could we do an `if else`?